src/init/features/dataconnect/index.ts

```diff
@@ -685,7 +677,7 @@ async function locationChoices(setup: Setup) {
  * Returns a unique ID that's either `recommended` or `recommended-{i}`.
  * Avoid existing IDs.
  */
-function newUniqueId(recommended: string, existingIDs: string[]): string {
+export function newUniqueId(recommended: string, existingIDs: string[]): string {
```

Comment:
Why was this exported?

Reply:
This is re-used in `sdk.ts` to find a unique new folder.

Probably should have extracted this to a helper package